TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/PageTests/Persons/PersonDetail/ChangeLogNpqTrnRequestSupportTaskUpdatedEventTests.cs

```diff
@@ -110,7 +110,7 @@ await CreateEvent(createdByUser.UserId, person.PersonId, applicationUser.UserId,
 
         var title = item.QuerySelector(".moj-timeline__title");
         Assert.NotNull(title);
-        Assert.Equal("Record updated from Apply for QTS TRN request", title.TrimmedText());
+        Assert.Equal("Record created from Apply for QTS TRN request", title.TrimmedText());
```

Comment:
because your test says this is about an update, the expectation is correct, but the data setup is wrong